solr/core/src/java/org/apache/solr/handler/admin/ClusterStatus.java

```diff
@@ -48,6 +48,49 @@
   private final ZkNodeProps message;
   private final String collection; // maybe null
 
+  /** Shard / collection health state. */
+  public enum Health {
+    /** All replicas up, leader exists. */
+    GREEN,
+    /** Some replicas down, leader exists. */
+    YELLOW,
+    /** Many replicas down, leader exists. */
+    ORANGE,
+    /** No leader or most replicas down. */
```

Comment:
With the latest changes, this will read "No leader or all replicas down". Also, the ORANGE level can be written as "Majority of replicas down, leader exists"?